ext/spl/tests/iterator_to_array.phpt

```diff
@@ -13,7 +13,7 @@ iterator_to_array('test','test');
 
 ?>
 --EXPECTF--
-Fatal error: Uncaught TypeError: iterator_to_array(): Argument #1 ($iterator) must be of type Traversable, string given in %s:%d
+Fatal error: Uncaught TypeError: iterator_to_array(): Argument #1 ($iterator) must be of type iterable, string given in %s:%d
```

Comment:
Huh, I would have expected that because of #7309 this would know be ``Traversable|array`` must have forgotten to change it in ZPP

Reply:
Okay this now needs to be adjusted as #8838 landed